src/app/routes/api/v3/admin/forms/__tests__/admin-forms.form.routes.spec.ts

```diff
@@ -2435,4 +2437,219 @@ describe('admin-form.form.routes', () => {
       )
     })
   })
+
+  describe('POST /admin/forms/feedback', () => {
```

Comment:
shall we add the test case for 422 as well?

Reply:
We no longer throw 422 as we rely on the authed session for userId, and will throw 404 if not found instead. 

Have updated the docs here [8405543](https://github.com/opengovsg/FormSG/pull/6454/commits/84055434476aa416b0af672dc402825f3c6ae14d)

Reply:
ok ya. i was seeing 422 below, that's why im asking this, thx for updating it.